README.md

```diff
@@ -27,13 +28,16 @@ Bring real-time observability data directly into your development workflow.
 
 You can add this MCP server (using STDIO) to your MCP Client like VS Code, Claude, Cursor, Amazon Q Developer CLI, Windsurf Github Copilot via the package `@dynatrace-oss/dynatrace-mcp-server`.
 
+We recommend to always set it up for your current workspace, instead of using it globally.
```

Comment:
I don't think a comma is syntactically required here, but it's not going to make or break things.

```suggestion
We recommend to always set it up for your current workspace instead of using it globally.
```